### PR TITLE
chore: refactor map alerts to use AlertStack and AlertBar from @dhis2/ui (DHIS2-9699)

### PR DESCRIPTION
**Pull request by turban**
Partly fixes: https://jira.dhis2.org/browse/DHIS2-9699

This PR refactors the alert/message system used in Maps. The main changes are: 
- Two alert systems (confirmation messages, and error/warning alerts) are merged into one. 
- We switch from MUI Snackbar to AlertStack and AlertBar from @dhis2/ui
- We now support showing multiple alerts at the same time. 
- We use the built-in alert styles from  @dhis2/ui (success, warning, critical). 
- Alerts are shown for 10 seconds (previously 5 seconds)

Screenshot 2020-11-10 at 14 06 00

After this PR: 

Screenshot 2020-11-10 at 14 13 41

Screenshot 2020-11-10 at 14 20 28

Screenshot 2020-11-10 at 14 26 34

### CODE REVIEW

File: i18n/en.pot [hunk 1]
```diff
@@ -1043,19 +1046,16 @@ msgid ""
 "Google. Please check the DHIS 2 documentation."
 msgstr ""
 
-msgid "Error"
-msgstr ""
-
 msgid "A connection to Google Earth Engine could not be established."
```
Comment (cooper-joe) [inline i18n/en.pot:1049]: Suggested text: "Cannot connect to Google Earth Engine."

File: src/epics/earthEngine.js [hunk 2]
```diff
@@ -148,14 +147,12 @@ export const loadCollection = action$ =>
...
-                        i18n.t(token.message),
-                        i18n.t(
-                            'To show this layer you must first sign up for the Earth Engine service at Google. Please check the DHIS 2 documentation.'
-                        )
-                    )
-                );
+                return setAlert({
+                    warning: true,
+                    message: i18n.t(
+                        'To show this layer you must first sign up for the Earth Engine service at Google. Please check the DHIS 2 documentation.'
```
Comment (cooper-joe) [inline src/epics/earthEngine.js:153]: Suggested content: "This layer requires a Google Earth Engine account. Check the DHIS2 documentation for more information".

File: src/loaders/eventLoader.js [hunk 3]
```diff
@@ -16,21 +16,20 @@ import {
...
-    i18n.t('Access denied'),
-    i18n.t('user is not allowed to read layer data')
-);
-const unknownErrorAlert = createAlert(
-    i18n.t('Failed'),
-    i18n.t('an unknown error occurred while reading layer data')
-);
+const accessDeniedAlert = {
+    warning: true,
+    message: i18n.t('Access denied - user is not allowed to read layer data'),
```
Comment (cooper-joe) [inline src/loaders/eventLoader.js:27]: Suggested content: "You don't have access to this layer data".

File: src/util/favorites.js [hunk 4]
```diff
@@ -231,7 +230,14 @@ export const translateConfig = config => {
             return {
                 el,
                 name,
-                alerts: [createAlert(name, i18n.t('Map could not be created'))],
+                alerts: [
+                    {
+                        critical: true,
+                        message: `${name}: ${i18n.t(
+                            'Map could not be created'
```
Comment (cooper-joe) [inline src/util/favorites.js:237]: I'm not sure on the context here, but is this for saving a Map? If so, I suggest the text: "There was a problem saving this Map. Changes have not been saved."

**Review by cooper-joe — APPROVED** [review]
Looks good! I've added some suggested for adjusting the content of alerts. These will require upstream translation changes, so I am not marking as requested changes -- they could be covered in a future PR if needed.

File: src/util/favorites.js (hunk 4 above)
Comment (turban) [reply src/util/favorites.js:237]: @cooper-joe, the message is shown if it was not possible to create a map from a chart or pivot.

File: i18n/en.pot (hunk 1 above)
Comment (turban) [reply i18n/en.pot:1049]: Fixed.

File: src/epics/earthEngine.js (hunk 2 above)
Comment (turban) [reply src/epics/earthEngine.js]: Fixed.

File: src/loaders/eventLoader.js (hunk 3 above)
Comment (turban) [reply src/loaders/eventLoader.js]: Fixed.

File: src/util/favorites.js (hunk 4 above)
Comment (cooper-joe) [reply src/util/favorites.js:237]: Oh right, thanks for clarifying. Then I think "There was a problem creating a map from the imported data." works well.

Comment (turban) [reply src/util/favorites.js]: Fixed.

**Pull request merged**